src/fileformats/ocd_file_export.cpp

```diff
@@ -2384,7 +2210,7 @@ QByteArray OcdFileExport::exportCombinedLineSymbol(
 void OcdFileExport::exportSymbolIcon(const Map* map, const Symbol* symbol, Ocd::IconV8& icon)
 {
 	// Icon: 22x22 with 4 bit palette color, origin at bottom left
-	constexpr int icon_size = 22;
+	constexpr int icon_size = Ocd::IconV8::size();
```

Comment:
Naming. When I see `size()` method on an image-like object, I expect it to return `QSize`. `width()` would tell me better that a single value is being returned.

Reply:
Okay, 'width' is also what I use in the GUI. 
(I want to avoid adding more Qt types in the Ocd namespace. Most types can be easily replaced with C++11 ones.)

Reply:
I'm sure we are on the same page wrt usage of Qt types in generic code. My remark was along the lines that when I, and maybe other programmers too, read code, it is easier to work with one concept, rather than switching between multiple ones. `size()` on two dimensional object immediately translates to width x height for me. The point is that having to remember that `Ocd::IconV*`'s `size()` is a scalar is an extra thing to keep in mind.